### ISSUE TITLE
Fail to reproduce the pretrained model using the DenseNet backbone.

### ISSUE THREAD

**Issue by starfudan2012** [LJOVO/TranSalNet#12]
Hi,

I trained the TranSalNet_Dense model from scratch using the default setup on the SALICON2017 dataset, but my results don't match the performance of the pretrained weights. Is this expected behavior?

<img width="841" alt="Image" src="https://github.com/user-attachments/assets/e38ddf23-a9bc-433d-84ed-43a447691f2f" />

Thanks

**Comment by LJOVO** [comment]
Hi,

This is possible, especially when training from scratch. 

Best

**Comment by starfudan2012** [comment]
Thanks for sharing. Regarding the objective metrics, the shared pretrained model performs much better (by about 5%) than the model trained from scratch. I’ve tried several runs, but none of them have reached the same quality as the pretrained model. Do you have any suggestions for achieving the same quality as the shared model?

**Comment by LJOVO** [comment]
Hi, thanks for the follow-up.

The model can be somewhat unstable to train. Based on my experience, here are a few suggestions that might be useful:
- Double-check the data processing pipeline, especially how the fixation maps are handled. 
- It is sensitive to randomness. Try running the experiment with a few different random seeds, as one may lead to a much better result.
- If the gap persists, consider fine-tuning the learning rate and weight decay.

Best

**Comment by starfudan2012** [comment]
These are very helpful. Thanks.
